### PR TITLE
Fix for link error in installation.md for ubuntu (fixes #1339)

### LINKED ISSUE #1339: Step 1- Installation in Ubuntu
In the Note section, the link to downloading the bash file for installation in ubuntu does not seem to be working. It is redirecting to "Sorry page does not exist". I think the path is wrong. It is directing to pages and then techgenius in the repository but there is no uploads in techgenius and the correct link directing to the bash file would be in the repo-pages-uploads-bashscripts-installations.h. Attached are the screenshots I took while clicking the page. Let me know if this is a valid issue or not. Thanks!!!
screenshot from 2017-10-26 08-41-30
screenshot from 2017-10-26 08-42-02

### PR DESCRIPTION
**Pull request by lorugant**
Rawgit link:

https://raw.githubusercontent.com/lorugant/lorugant.github.io/lorugantbranch/pages/techgenius/tg-installation.md

Please let me know if it is the right way to solve. Thanks!!

### CODE REVIEW

File: pages/techgenius/tg-installation.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Open your `Terminal`. We assume that [brew](http://brew.sh/) is already installe
 
 ## Ubuntu
 
-Note: You may [download this bash file](uploads/bashScripts/instructions.sh) to automate the installation process. Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
+Note: You may [download this bash file](https://github.com/open-learning-exchange/open-learning-exchange.github.io/blob/master/pages/uploads/bashScripts/instructions.sh) to automate the installation process. The bash file can be downloaded by right clicking on the Raw button and clicking on "save link as" present in the top right corner.Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
```
Comment (xyb994) [inline pages/techgenius/tg-installation.md]: We want to use relative links as much as possible in our MDwiki. 

Reference: https://github.com/open-learning-exchange/open-learning-exchange.github.io/issues/515

---

Aside from that, I'm not sure if this installation script section is necessary. It's only 6 lines. Plus there's no indication that this script covers the rest of the page's content. I feel `macOS`, `Ubuntu`, and `macOS and Ubuntu ONLY` kinda cluttered together, they could be reorganized better.

Comment (dogi) [reply pages/techgenius/tg-installation.md]: please use no github link but relative one for the first link ...

File: pages/techgenius/tg-installation.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ Open your `Terminal`. We assume that [brew](http://brew.sh/) is already installe
 
 ## Ubuntu
 
-Note: You may [download this bash file](uploads/bashScripts/instructions.sh) to automate the installation process. Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
+Note: You may [download this bash file](uploads/bashScripts/instructions.sh) to automate the installation process. The bash file can be downloaded by right clicking on the Raw button and clicking on "save link as" present in the top right corner.Make sure that the bash file is in the directory where you would like to store the [ole--vagrant-vi](https://github.com/dogi/ole--vagrant-vi) repository. Go to terminal and use command `sudo sh instructions.sh`
```
Comment (kylemathias) [inline pages/techgenius/tg-installation.md]: I think @dogi wanted you to change the link (https://github.com/dogi/ole--vagrant-vi) to a relative link. A example would be the link right above "download this bash file".

I think the problem is that (https://github.com/dogi/ole--vargrant-vi) is in @dogi repository only. correct? so the way you have it now should be fine.

**Comment by xyb994** [comment]
Closing this since the installation script for Ubuntu is now removed.

**Pull request closed without merging**